sidekiq/README.md

```diff
@@ -57,27 +57,10 @@ No additional installation is needed on your server.
        mappings:
          - match: "sidekiq.sidekiq.*"
            name: "sidekiq.*"
-         - match: "sidekiq.jobs.*.perform.avg"
-           name: "sidekiq.jobs.perform.avg"
+         - match: "sidekiq.jobs.*.perform"
+           name: "sidekiq.jobs.perform"
```

Comment:
1. This won't match anymore:
- `sidekiq.jobs.<JOB_NAME>.perform.count`
- `sidekiq.jobs.<JOB_NAME>.perform.avg`
- `sidekiq.jobs.<JOB_NAME>.perform.median`
- etc

2. It seems that job name might contain `.`, in that case, we need to use https://docs.datadoghq.com/developers/dogstatsd/dogstatsd_mapper/#regex-match-pattern

Reply:
Adressed in https://github.com/DataDog/integrations-core/pull/6697